.travis/build.sh

```diff
@@ -43,7 +47,7 @@ for i in $($GNU_FIND -name *.testout); do
 done
 
 #exit with exit code 1 if any test failed
-if [ -n "$($GNU_FIND -name '*.testout')" ]; then
+if [ -n "$($GNU_FIND -name '*.testout')" -o -n "$DEMO_FAILED" ]; then
```

Comment:
Ah I forgot that I chose not to delete the testout when failure occurs. You are right, and I will fix.